x-pack/plugins/actions/common/routes/connector/response/schemas/v1.ts

```diff
@@ -8,34 +8,89 @@
 import { schema } from '@kbn/config-schema';
 
 export const connectorResponseSchema = schema.object({
-  id: schema.string(),
-  name: schema.string(),
+  id: schema.string({
+    meta: {
+      description: 'The identifier for the connector.',
+    },
+  }),
+  name: schema.string({
+    meta: {
+      description: ' The name of the rule.',
+    },
+  }),
   config: schema.maybe(schema.recordOf(schema.string(), schema.any())),
-  connector_type_id: schema.string(),
-  is_missing_secrets: schema.maybe(schema.boolean()),
-  is_preconfigured: schema.boolean(),
-  is_deprecated: schema.boolean(),
-  is_system_action: schema.boolean(),
+  connector_type_id: schema.string({
+    meta: { description: 'The connector type identifier.' },
+  }),
+  is_missing_secrets: schema.maybe(
+    schema.boolean({ meta: { description: 'Indicates whether the connector is missing secrets.' } })
+  ),
+  is_preconfigured: schema.boolean({
+    meta: { description: 'Indicates whether the connector is preconfigured.' },
+  }),
+  is_deprecated: schema.boolean({
+    meta: { description: 'Indicates whether the connector is deprecated.' },
+  }),
+  is_system_action: schema.boolean({
+    meta: { description: 'Indicates whether the connector is system action.' },
+  }),
 });
 
 export const allConnectorsResponseSchema = connectorResponseSchema.extends({
-  referenced_by_count: schema.number(),
+  referenced_by_count: schema.number({
+    meta: {
+      description: ' The number of rules that are currently using the connector.',
```

Comment:
In the manually maintained file we had this info as well:
```suggestion
      description: 'The number of rules that are currently using the connector. If `is_preconfigured` is true, this value is not calculated.',
```
... We also said it's the "number of saved objects that reference the connector". If that's no longer true and it tracks only the rules now that's fine, just wanted to call out the disparity.

Reply:
Oh okay, I think that is still true

Reply:
Should I change this to be
The number of saved objects that reference the connector. If `is_preconfigured` is true, this value is not calculated.

Reply:
That LGTM, thanks!

Reply:
Resolved in this commit [f9a331f](https://github.com/elastic/kibana/pull/192767/commits/f9a331f2df86140292b0ef8b5766aa2530536339)